sidecar-injector/bin/publish

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash -e
+#
+# Build and push secretless sidecar injector mutating webhook service
+# usage: ./bin/publish
+set -ex
+
+cd $(dirname $0)/..
```

Comment:
use the new helper `abspath` - you can see how this is done in other scripts

Reply:
I did see that but since this is not in `$ROOT_DIR/bin` then using `abspath` seemed like more work than what I've done. Also this is bound to be split out.